msteams-platform/toolkit/create-new-project.md

```diff
@@ -6,40 +6,42 @@ ms.author: surbhigupta
-
-::: zone pivot="visual-studio-code"
+In this section, you can learn how to create a new Teams project using Microsoft Visual Studio Code.
 
 ## Create a new Teams project for Visual Studio Code
 
-You can build a new Teams project by selecting **Create a new Teams app** in Teams Toolkit. You can create the following types of apps in Teams Toolkit:
+You can build a new Teams project by selecting **Create a New App** in Teams Toolkit. You can start from built-in Teams app templates or start from official Teams app samples in Teams Toolkit. What's more, Teams Toolkit V5.0.0 supports to start with Outlook Add-ins templates to build your own Outlook Add-ins.
+
+:::image type="content" source="../assets/images/teams-toolkit-v2/first-tab/create-new-app.PNG" alt-text="Screenshot shows the Create a New App dropdown.":::
+
+To start with Teams capabilities, you can create the following types of Teams app:
 
 | App Types | Definition |
 | --- | --- |
-| **Basic Teams apps** | Basic Teams apps are tab, bot, or message extension that you can create and customize based on your requirement. |
-| **Scenario-based Teams apps** | Scenario-based Teams apps are notification bot, command bot, workflow bot, SSO-enabled tab, or SPFx tab app and these are suitable for one particular scenario. For example, a notification bot is suitable to send notifications and not used for chat. |
+| **Scenario-based Teams apps** | This group of templates are designed for particular abstracted business scenarios that your teams app can serve for. For example notification bot, command bot, SSO-enabled tab, or Dashboard tab app. |
+| **Basic Teams apps** | Basic Teams apps are just hello world Teams tab, bot, or message extension that you can create and customize based on your requirement. |
+| **Extend Teams App across Microsoft 365** | This group of Teams app can be installed and run on Outlook and Office.com. |
 
 ## Create a new Teams app
 
-The process to create a new Teams app is similar for all types of apps except SPFx tab app, workflow bot, and notification bot.
+The process to create a new Teams app is similar for all types of apps.
 
 **To create a basic Teams app**:
 
 1. Open **Visual Studio Code**.
 
-1. Select the **Teams Toolkit** > **Create a new Teams app**.
+1. Select the **Teams Toolkit** > **Create a New App**.
 
     :::image type="content" source="../assets/images/teams-toolkit-v2/first-tab/create-project.png" alt-text="screenshot shows the Create New Project button in the Teams Toolkit sidebar.":::
 
 1. Select **Start with a Teams capability**.
 
-    :::image type="content" source="../assets/images/teams-toolkit-v2/first-tab/select-app-type.png" alt-text="Screenshot shows the option to select app type." lightbox="../assets/images/teams-toolkit-v2/first-tab/select-app-type.png":::
+    :::image type="content" source="../assets/images/teams-toolkit-v2/first-tab/create-new-app.PNG" alt-text="Screenshot shows the Create a New App dropdown.":::
```

Comment:
now we don't have start with Teams capability, so can remove this step. Instead, we direct ask user to select from "tab" or "bot"

Reply:
need new screenshot too.

Reply:
Updated.